soc/st/stm32/stm32n6x/CMakeLists.txt

```
@@ -24,6 +24,7 @@ if(SIGNING_TOOL_FIND STREQUAL SIGNING_TOOL_FIND-NOTFOUND)
   Refer to board documentation for more information")
 else()
   set_property(GLOBAL APPEND PROPERTY extra_post_build_commands
+    COMMAND rm -f ${PROJECT_BINARY_DIR}/${CONFIG_KERNEL_BIN_NAME}.signed.bin
```

`rm` is not portable, use CMake in command mode instead:
```suggestion
    COMMAND ${CMAKE_COMMAND} -E rm -f ${PROJECT_BINARY_DIR}/${CONFIG_KERNEL_BIN_NAME}.signed.bin
```

---

Done